src/app/services/code-view/angular-code-service.ts

```diff
@@ -226,11 +235,24 @@ export class AngularCodeService extends CodeService {
             if(sampleContent.addTsConfig) {
                 codeService.sharedFileContent.files.push(codeService.sharedFileContent.tsConfig)
             }
-            let formData = {
-                dependencies: sampleContent.sampleDependencies,
-                files: codeService.sharedFileContent.files.concat(sampleContent.sampleFiles),
-                devDependencies: codeService.sharedFileContent.devDependencies
+
+            let formData = {};
+
+            if(util.isLocalhost && codeService.localDemosUrls > 1){
+                const key :any = $codeView[0].dataset["demosBaseUrl"];
```

Comment:
You can get the baseUrl from the code view with the demosBaseUrlAttrName attribute of the code view